pkg/api/api.go

```diff
@@ -0,0 +1,89 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/nukleros/eks-cluster/pkg/resource"
+)
+
+var (
+	configFile string
+)
+
+// Create creates an EKS cluster, writes the inventory file, and cleans up resources in
+// the event of a failure
```

Comment:
Missing the period at the end of this sentence.  The same applies to some other comments in this file.  (Recognizing this may have been my mistake that was copied over.)